go/vt/vtgate/planbuilder/show.go

```diff
@@ -641,6 +644,26 @@ func buildEnginesPlan() (engine.Primitive, error) {
 		buildVarCharFields("Engine", "Support", "Comment", "Transactions", "XA", "Savepoints")), nil
 }
 
+func buildVschemaKeyspacesPlan(vschema plancontext.VSchema) (engine.Primitive, error) {
+	vs := vschema.GetVSchema()
+	var rows [][]sqltypes.Value
+	for ksName, ks := range vs.Keyspaces {
+		var row []sqltypes.Value
+		row = append(row, sqltypes.NewVarChar(ksName))
+		row = append(row, sqltypes.NewVarChar(strconv.FormatBool(ks.Keyspace.Sharded)))
+		fkMode, _ := vschema.ForeignKeyMode(ksName)
+		row = append(row, sqltypes.NewVarChar(fkMode.String()))
+		ksError := ""
+		if ks.Error != nil {
+			ksError = ks.Error.Error()
+		}
+		row = append(row, sqltypes.NewVarChar(ksError))
+		rows = append(rows, row)
+	}
+
+	return engine.NewRowsPrimitive(rows, buildVarCharFields("Keyspace Name", "Sharded", "Foreign Key Mode", "Error")), nil
```

Comment:
```suggestion
	return engine.NewRowsPrimitive(rows, buildVarCharFields("Keyspace", "Sharded", "Foreign Key", "Comment")), nil
```

`Comment` will be more generic and can contain the error. Possibly can be used for additional information.

Reply:
Done